I18N.md

```diff
@@ -0,0 +1,55 @@
+## i18n
+
+See [i18n](link) for information on our internationalization tools and guidelines
+
+We use [react-i18next](https://react.i18next.com/) to translate messages in assisted-ui-lib. The
+i18n namespace must be declared in TRANSLATION_NAMESPACE property to avoid naming conflicts.
+
+- Environment variables:
+
+```
+  TRANSLATION_NAMESPACE: required, name of translation namespace
+```
+
+To test to standalone we need to put TRANSLATION_NAMESPACE=cim_namespace
+
+- We create the wrapper src/common/i18n/cim-18n that use the `useTranslation` hook with our
+  namespace as follows:
+
+```tsx
+import { useTranslation as useReactI18NextTranslation } from 'react-i18next';
+
+export function useTranslation() {
+  return useReactI18NextTranslation(`${process.env.TRANSLATION_NAMESPACE}`);
+}
+```
```

Comment:
I think we dont want to copy-paste the implementation of our `useTranslation` hook as it can change often. Better would be to add a link here